src/packages/sdk/src/utils/parseEnvValue.ts

```diff
@@ -1,26 +1,55 @@
-import { EnvValue } from '@prisma/generator-helper'
+import { EnvValue, BinaryTargetsEnvValue } from '@prisma/generator-helper'
 import chalk from 'chalk'
 
 /**
- * Parses the EnvValue and return the string value
+ * Parses an EnvValue (example: Provider) and return the string value
  *
  * - If there is no env var just return the value
  * - If there is an env var it will be resolve and returned.
  * - If there is an env var is present but can't be resolved an error will be thrown
  */
-export function parseEnvValue(provider: EnvValue) {
-  if (provider.fromEnvVar && provider.fromEnvVar !== 'null') {
-    const value = process.env[provider.fromEnvVar]
+export function parseEnvValue(object: EnvValue) {
+  if (object.fromEnvVar && object.fromEnvVar !== null) {
+    const value = process.env[object.fromEnvVar]
     if (!value) {
       throw new Error(
         `Attempted to load provider value using \`env(${
-          provider.fromEnvVar
+          object.fromEnvVar
         })\` but it was not present. Please insure that ${chalk.dim(
-          provider.fromEnvVar,
+          object.fromEnvVar,
         )} is present in your Environment Variables`,
       )
     }
+
     return value
   }
-  return provider.value
+
+  return object.value
+}
+
+/**
+ * Parses the BinaryTargetsEnvValue and return the string value
+ *
+ * - If there is no env var just return the value
+ * - If there is an env var it will be resolve and returned.
+ * - If there is an env var is present but can't be resolved an error will be thrown
+ */
+export function parseBinaryTargetsEnvValue(object: BinaryTargetsEnvValue) {
+  if (object.fromEnvVar && object.fromEnvVar !== null) {
+    const value = process.env[object.fromEnvVar]
+    if (!value) {
+      throw new Error(
+        `Attempted to load binaryTargets value using \`env(${
+          object.fromEnvVar
+        })\` but it was not present. Please insure that ${chalk.dim(
```

Comment:
"ensure" instead of "insure", you probably ctrl-c/v from above

Reply:
Good point!